lib/local-links-manager/import/analytics_importer.rb

```diff
@@ -26,7 +26,7 @@ def import_item(item, _response, summariser)
         link = Link.lookup_by_base_path(item[:base_path])
 
         if link
-          link.update!(analytics: item[:clicks])
+          link.update!(analytics: item[:clicks] || 0)
```

Comment:
Just had a thought that it might be nicer to do this:

```suggestion
          link.update!(analytics: item.fetch(:clicks, 0))
```

But this depends on whether `:clicks` exists in `item` but is `nil`, or if it doesn't exist in `item`. The solution you've already got using `||` handles both cases, so perhaps it's okay to stick with this one.

Reply:
I'd personally feel it's better to leave both cases handled, given I'm still not 100% how GA handles this figure.